backend/src/main/java/backend/backend/exception/globalcontrolleradvice/ControllerAdvice.java

```diff
@@ -24,9 +24,16 @@ public ExceptionMessage notFoundException(NotFoundException e) {
         return new ExceptionMessage(e.getMessage());
     }
 
+    @ExceptionHandler(InvalidVerificationCodeException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ExceptionMessage invalidVerificationCodeException(InvalidVerificationCodeException e) {
+        log.info("InvalidVerificationCodeException: {}", e.getMessage(), e);
```

Comment:
예외정도면
```suggestion
        log.warn("InvalidVerificationCodeException: {}", e.getMessage(), e);
```
warn이 적당할 수도 있어요!

logging level에 대해 고민해보고 팀원과 공유해봐요!
더 나아가서 루나팀내에서 공유된다면 더 좋을 것 같아요!